WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Grouped Products/ProductListMultiSelectorSearchUICommand.swift

```diff
@@ -0,0 +1,123 @@
+import Yosemite
+
+/// Implementation of `SearchUICommand` for selecting linked products to a grouped product from search UI.
+final class ProductListMultiSelectorSearchUICommand: SearchUICommand {
+    typealias Model = Product
+    typealias CellViewModel = ProductsTabProductViewModel
+    typealias ResultsControllerModel = StorageProduct
+
+    let searchBarPlaceholder = NSLocalizedString("Search all products", comment: "Products Search Placeholder")
+
+    let searchBarAccessibilityIdentifier = "product-search-screen-search-field"
+
+    let cancelButtonAccessibilityIdentifier = "product-search-screen-cancel-button"
```

Comment:
good question, these two accessibility identifiers (`searchBarAccessibilityIdentifier` and `cancelButtonAccessibilityIdentifier`) are actually part of the `SearchUICommand` protocol to enforce accessibility support:

https://github.com/woocommerce/woocommerce-ios/blob/9a0409d302549e9548744ef8c138f739301e284f/WooCommerce/Classes/ViewRelated/Search/SearchUICommand.swift#L74-L78

otherwise, we usually keep them as private extensions in the app :)